lib/travis/logs/helpers/database.rb

```diff
@@ -54,7 +54,7 @@ def self.jdbc_uri_from_config(config)
           params.merge!(
             ssl: true,
             sslfactory: 'org.postgresql.ssl.NonValidatingFactory'
-          ) unless %w(1 yes on).include?(ENV['PG_DISABLE_SSL'].to_s.downcase)
+          ) if config[:ssl]
```

Comment:
shouldn't this be using https://github.com/travis-ci/travis-logs/pull/43/files#diff-13fa1867dbc7ecbff0f1d8be526dc13dR7 or https://github.com/travis-ci/travis-logs/pull/43/files#diff-13fa1867dbc7ecbff0f1d8be526dc13dR11 ?